src/components/PageWizard/PageWizard.jsx

```diff
@@ -1,137 +1,28 @@
-        <Button onClick={onBack} kind="secondary">
-          {i18n.back}
-        </Button>
-      ) : null}
-      {hasNext ? (
-        <Button onClick={() => onValidate() && onNext()} disabled={nextDisabled}>
-          {i18n.next}
-        </Button>
-      ) : (
-        <Button
-          onClick={() => onValidate() && onSubmit()}
-          disabled={nextDisabled}
-          loading={sendingData}
-        >
-          {i18n.submit}
-        </Button>
-      )}
-    </div>
-  </div>
-);
-PageWizardStep.propTypes = {
-  id: PropTypes.string.isRequired, // eslint-disable-line
-  onValidate: PropTypes.func, // eslint-disable-line
-  children: childrenPropType,
-  /** optional error to show for this step */
-  error: PropTypes.node,
-  /** optional callback to clear the error */
-  onClearError: PropTypes.func,
-  /** Internationalized strings */
-  i18n: PropTypes.objectOf(PropTypes.string).isRequired,
-  hasStickyFooter: PropTypes.bool,
-  onClose: PropTypes.func.isRequired,
-  hasPrev: PropTypes.bool.isRequired,
-  hasNext: PropTypes.bool.isRequired,
-  onNext: PropTypes.func.isRequired,
-  nextDisabled: PropTypes.bool,
-  onBack: PropTypes.func.isRequired,
-  sendingData: PropTypes.bool,
-  onSubmit: PropTypes.func.isRequired,
-};
-PageWizardStep.defaultProps = {
-  children: [],
-  error: null,
-  onClearError: null,
-  onValidate: () => true,
-  hasStickyFooter: false,
-  nextDisabled: false,
-  sendingData: false,
-};
-
-const PageWizardStepTitle = ({ children }) => (
-  <div className="page-wizard--step--title">{children}</div>
-);
-PageWizardStepTitle.propTypes = { children: childrenPropType };
-PageWizardStepTitle.defaultProps = { children: [] };
-
-const PageWizardStepDescription = ({ children }) => (
-  <div className="page-wizard--step--description">{children}</div>
-);
-PageWizardStepDescription.propTypes = { children: childrenPropType };
-PageWizardStepDescription.defaultProps = { children: [] };
+export const childrenPropType = PropTypes.oneOfType([
+  PropTypes.arrayOf(PropTypes.node),
+  PropTypes.node,
+]);
 
-const PageWizardStepContent = ({ children }) => (
-  <div className="page-wizard--step--content">{children}</div>
-);
-PageWizardStepContent.propTypes = { children: childrenPropType };
-PageWizardStepContent.defaultProps = { children: [] };
-
-const PageWizardStepExtraContent = ({ children }) => (
-  <div className="page-wizard--step--extra-content">{children}</div>
-);
-PageWizardStepExtraContent.propTypes = { children: childrenPropType };
-PageWizardStepExtraContent.defaultProps = { children: [] };
-
-export const propTypes = {
+export const PageWizardPropTypes = {
   children: childrenPropType,
   /** Id of current step */
   currentStepId: PropTypes.string,
   /** action when click next button called with no param */
-  onNext: PropTypes.func,
+  onNext: PropTypes.func.isRequired,
   /** action when click back button called with no param */
-  onBack: PropTypes.func,
+  onBack: PropTypes.func.isRequired,
```

Comment:
it might be a mistake to make this required as they support single step page wizards.  The same is true for onNext and setStep

Reply:
good catch Scott

Reply:
Good catch, removed.